docs/API.md

```diff
@@ -1395,12 +1422,31 @@ Example: `GET trusty/wordpress-42/meta/revision-info`
 }
 ```
 
+Example: `GET wordpress/meta/revision-info`
+
+```json
+{
+    "Revisions": [
+        "cs:trusty/wordpress-43",
+        "cs:trusty/wordpress-42",
+        "cs:trusty/wordpress-41",
+        "cs:trusty/wordpress-39"
+        "cs:precise/wordpress-41",
+        "cs:precise/wordpress-39"
+    ]
+}
+```
+
 #### GET *id*/meta/id
 
 The `id` path returns information on the charm or bundle id, split apart into
 its various components, including the id itself. The information is exactly
 that contained within the entity id.
 
+The request URL may omit series. For charms which support multiple series
```

Comment:
The first sentence is redundant - the series may always be omitted from an id.

Returning the preferred series when there are multiple supported series may be problematic. Currently it is an invariant that a resolved URL (which is exactly what meta/id is returning) refers directly to entity entry in the charm store. Changing this may break quite a bit of code.

Ideally I'd like the API to change so that a resolved id for a multi-series charm does not contain the series, but this could be considered a backwardly incompatible change, and old clients will then refuse to deploy multi-series charms.

One possibility is to create a new version of the API (ideally by copying the entire API implementation into v5 and creating v4 layering on top of v5), but this would require some thought.